tests/test_references.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from vulndb import DBVuln
+from tests.vulndb_test import VulnDBTest
+
+class TestReferences(VulnDBTest):
+
+    def test_no_redundant_cve_mitre_org_urls(self):
+        invalid = []
```

Comment:
To make this 100% complete please do add a doc to the test method that explains why this is bad, and what the developer should do to fix it.